ios-framework/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import io.github.droidkaigi.feeder.Dep
+
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+}
+
+apply(rootProject.file("gradle/android.gradle"))
+
+kotlin {
+    android()
+
+    val iosTarget: (String, KotlinNativeTarget.() -> Unit) -> KotlinNativeTarget =
+        if (System.getenv("SDK_NAME")?.startsWith("iphoneos") == true)
+            ::iosArm64
+        else
+            ::iosX64
+
+    iosTarget("ios") {
+        binaries {
+            framework {
+                baseName = "Shared"
```

Comment:
Thank you for creating iOS combine framework! 👍 
How about "DroidKaigiMPP" for framework name 👀

Reply:
Sorry, I changed my name.
I'll change it back!

https://github.com/DroidKaigi/conference-app-2021/blob/484e986ba39a23650e1a1e0729509a6af67a0117/ios/project.yml#L52-L53

Reply:
Fixed 56e3f27 🙇‍♂️